src/main/java/uk/gov/companieshouse/insolvency/data/api/InsolvencyApiService.java

```diff
@@ -80,6 +87,15 @@ private ChangedResource mapChangedResource(String contextId,
         changedResource.setResourceKind("company-insolvency");
         changedResource.setContextId(contextId);
 
+        if (isDelete) {
+            try {
+                changedResource.setDeletedData(objectMapper.writeValueAsString(insolvencyDocument));
+            } catch (JsonProcessingException exp) {
+                logger.error("Error occurred while serializing to json", exp);
+                throw new RuntimeException(exp);
```

Comment:
Should this be some kind of retryable error?

Reply:
@zahidm3 what do you say about this?